### ISSUE TITLE
[MASSEMBLY-927] Support for properties mapping on executions of maven-assembly-plugin

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-assembly-plugin#1140]
**[Darío Cutillas](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=dacucar)** opened **[MASSEMBLY-927](https://issues.apache.org/jira/browse/MASSEMBLY-927?redirect=false)** and commented

Support `<properties>` optional parameter on the `single` goal.

The maven-assembly-plugin supports the use of `<filters>` to specify files with additional properties to use during the filtering.

I'd like to have the possibility of specifying a set properties in the pom without having to create separate property files.

P.g.:

```xml
<configuration>
    <properties>
        <myproperty>value</myproperty>
    </properties>
</configuration>
```

---
No further details from [MASSEMBLY-927](https://issues.apache.org/jira/browse/MASSEMBLY-927?redirect=false)

**Comment by jira-importer** [comment]
**[Robert Scholte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=rfscholte)** commented

Fixed in [0a1aaada8a71334e1256587f91792b9807ae1b94](https://gitbox.apache.org/repos/asf?p=maven-assembly-plugin.git;a=commit;h=0a1aaada8a71334e1256587f91792b9807ae1b94)
The parameter `additionalProperties` has been added.

**Comment by jira-importer** [comment]
**[Darío Cutillas](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=dacucar)** commented

Thanks!
